client/components/content-elements/tce-api-video/server/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const { afterLoaded, afterSave, beforeSave } = require('tce-apivideo/dist/server/index.js');
```

Comment:
Can we import the whole module (assuming it only exports hooks) and then destructure it inside of exported object?
I.e.
```suggestion
const server = require('tce-apivideo/dist/server');
```